dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/ConsoleFormatTest.java

```diff
@@ -444,10 +500,10 @@ private static boolean isStringinSimpleFormat(String text) {
         return Pattern.compile(SIMPLE_FORMAT_REGEX_PATTERN).matcher(text).find();
     }
 
-    private void setInBootstrapPropertiesFile(RemoteFile bootstrapFile, String key, String value) throws Exception {
+    private void setInBootstrapPropertiesFile(LibertyServer Server, RemoteFile bootstrapFile, String key, String value) throws Exception {
```

Comment:
The variable name "Server" should be in lowercase, to meet java standards. Maybe, rename it to something else, so it doesn't conflict with "server"?